test/WebSites/TagHelpersWebSite/Views/Home/Environment.cshtml

```diff
@@ -0,0 +1,42 @@
+﻿@addtaghelper "Microsoft.AspNet.Mvc.TagHelpers"
+
+<h2>Environment Tag Helper Test</h2>
+
+<ul>
+    <environment names="Development">
+        <li>This will show</li>
+    </environment>
+    <environment names="development">
+        <li>This will show</li>
+    </environment>
+    <environment names="Development,Production">
+        <li>This will show</li>
+    </environment>
+    <environment names="Production,Development">
+        <li>This will show</li>
+    </environment>
+    <environment names="Local,Development,Production">
+        <li>This will show</li>
+    </environment>
+    <environment names="Development,">
+        <li>This will show</li>
+    </environment>
+    <environment>
+        <li>This will show</li>
+    </environment>
+    <environment names="">
+        <li>This will show</li>
+    </environment>
+    <environment names=" ">
+        <li>This will show</li>
+    </environment>
+    <environment names=", ">
```

Comment:
leave the bogus cases out of the functional tests.  do those in unit tests and stick w/ mainline scenarios here.